linear-algebra/src/main/scala/scalan/linalgebra/Matrices.scala

```diff
@@ -24,14 +32,18 @@ trait Matrices extends Vectors { self: MatricesDsl =>
     def apply(row: Rep[Int]): Vec[T]
     def apply(row: Rep[Int], column: Rep[Int]): Rep[T]
 
-    //def mapBy[R: Elem](f: Rep[T => R @uncheckedVariance]): Matrix[R]
-    def mapBy[R: Elem](f: Rep[Vector[T] => Vector[R] @uncheckedVariance]): Matr[R]
+    //def mapBy[R: Elem](f: Rep[T => R @uncheckedVariance]): Matr[R]
```

Comment:
Sorry, I didn't get it. We have `mapRowsBy` and I'd like to add `mapColumnsBy` for matrix traversal. `mapBy` is confusing IMO as it might imply mapping matrix elements, not rows or columns.